source/extensions/extensions_build_config.bzl

```diff
@@ -425,6 +425,12 @@ EXTENSIONS = {
 
     "envoy.rbac.matchers.upstream_ip_port":     "//source/extensions/filters/common/rbac/matchers:upstream_ip_port_lib",
 
+    #
+    # RBAC URI template matcher
+    #
+
+    "envoy.rbac.uri_template.uri_template_matcher":         "//source/extensions/filters/common/rbac/uri_template:uri_template_lib",
```

Comment:
There's no need to declare a new matcher here, we can use the existing URI template path matcher extension?

Reply:
If I understand correctly, this is necessary if I want to use a different factory for rbac, the one that was originally implemented with URI template path matcher returns [absl::StatusOr\<Router::PathMatcherSharedPtr>](https://github.com/envoyproxy/envoy/blob/main/source/extensions/path/match/uri_template/config.h#L19-L23), while the type needed in RBAC is [MatcherConstSharedPtr](https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/common/rbac/matchers.cc#L15-L16)

would it be better to use the same factory, but then wrap the `PathMatcherSharedPtr` into `MatcherConstSharedPtr`?